### ISSUE TITLE
Path for php_cs file not properly quoted

### ISSUE THREAD

**Issue by auroraeosrose** [EdgedesignCZ/phpqa#184]
Related to fix for  issue #180

The path passed to the command line is not properly quoted when passed to the command line, so if there are spaces in the path the tool will error

in .phpqa.yml
```
php-cs-fixer:
    config: .php_cs
    rules: '@PSR2'
    allowRiskyRules: false
    isDryRun: false
    verbose: true
```
Output

```
[Edge\QA\Task\ParallelExec] Output for "C:\xxxx/vendor/bin/php-cs-fixer" fix "./" --verbose --format junit --config C:\xxx\source\repos\Repo Name\tests\.php_cs:

 [Edge\QA\Task\ParallelExec]  In ConfigurationResolver.php line 570:                                                                      
 [PhpCsFixer\ConfigurationException\InvalidConfigurationException (16)]  
Cannot read config file "C:\xxx\source\repos\Repo"
```

I'm reverting back to a previous version of the tool for now since this is breaking my ci :(

**Comment by zdenekdrahos** [comment]
Fixed in [1.23.1](https://github.com/EdgedesignCZ/phpqa/releases/tag/v1.23.1)

**Comment by auroraeosrose** [comment]
thanks!
